docs/develop/typescript/failure-detection.mdx

```diff
@@ -1,164 +1,152 @@
 - [`startToCloseTimeout`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions/#starttoclosetimeout)
 - [`scheduleToStartTimeout`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions/#scheduletostarttimeout)
 
 ```typescript
-// Sample of typical options you can set
-const { greet } = proxyActivities<typeof activities>({
+const { myActivity } = proxyActivities<typeof activities>({
   scheduleToCloseTimeout: '5m',
   // startToCloseTimeout: "30s", // recommended
   // scheduleToStartTimeout: "60s",
-
-  retry: {
-    // default retry policy if not specified
-    initialInterval: '1s',
-    backoffCoefficient: 2,
-    maximumAttempts: Infinity,
-    maximumInterval: 100 * initialInterval,
-    nonRetryableErrorTypes: [],
-  },
 });
 ```
 
-### How to set an Activity Retry Policy {#activity-retries}
+## Activity Retry Policy {#activity-retries}
+
+**How to set an Activity Retry Policy using the Temporal TypeScript SDK**
 
 A Retry Policy works in cooperation with the timeouts to provide fine controls to optimize the execution experience.
 
 Activity Executions are automatically associated with a default [Retry Policy](/encyclopedia/retry-policies) if a custom one is not provided.
 
-To set Activity Retry Policies in TypeScript, pass [`ActivityOptions.retry`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions#retry) to [`proxyActivities`](https://typescript.temporal.io/api/namespaces/workflow/#proxyactivities).
+To set an Activity's Retry Policy in TypeScript, use the [`ActivityOptions.retry`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions#retry) property when creating the corresponding Activity proxy function using the [`proxyActivities()`](https://typescript.temporal.io/api/namespaces/workflow#proxyactivities) API.
 
 ```typescript
-// Sample of typical options you can set
-const { yourActivity } = proxyActivities<typeof activities>({
+const { myActivity } = proxyActivities<typeof activities>({
   // ...
   retry: {
-    // default retry policy if not specified
-    initialInterval: '1s',
-    backoffCoefficient: 2,
-    maximumAttempts: Infinity,
-    maximumInterval: 100 * initialInterval,
-    nonRetryableErrorTypes: [],
+    initialInterval: '10s',
+    maximumAttempts: 5,
   },
 });
 ```
 
-## How to Heartbeat an Activity {#activity-heartbeats}
+## Activity next Retry delay {#activity-next-retry-delay}
+
+**How to override the next Retry delay following an Activity failure using the Temporal TypeScript SDK**
+
+The time to wait after a retryable Activity failure until the next retry is attempted is normally determined by that Activity's Retry Policy.
+However, an Activity may override that duration when explicitly failing with an [`ApplicationFailure`](https://typescript.temporal.io/api/classes/common.ApplicationFailure) by setting a next Retry delay.
```

Comment:
I might suggest linking to "Application Failure" and the overview docs instead.

https://docs.temporal.io/references/failures#application-failure

That contains more broadly instructive information than the typescript one, while linking to typescript, and unfortunately AFAICT the typescript docs don't contain an uplink to the reference.